src/bidiMapper/domains/context/cdpTarget.ts

```diff
@@ -154,14 +154,19 @@ export class CdpTarget {
   }
 
   #setEventListeners() {
-    this.#cdpClient.on('*', (cdpMethod, params) => {
+    this.#cdpClient.on('*', (event, params) => {
+      // We may encounter uses for EventEmitter
+      // other then CDP events we want to skip them
```

Comment:
could you be more specific about what uses and why we use the event check for that?

Reply:
In Puppeteer we inject our own CDP implementation that uses their event emitter. On the connection part we send a  `this.emit(ConnectionEmittedEvents.Disconnected);` this is a Symbol that tries to be converted to String in the literal and fails. An another option would be to use `cdp.${String(event))`